keyboards/xd75/keymaps/msiu/rules.mk

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
This file can be deleted.

The code block here is left over from a previous build script, and is no longer needed.